### ISSUE TITLE
Content of error message is miss-spelled

### ISSUE THREAD

**Issue by AndreasTWolf** [servicecatalog/development#339]
[Description]
The error message, which occurs if the Stack name contains in valid characters, is using wrong terms for special characters

[Steps to reproduce]
Install Open-Stack Adapter;
Subscribe to OpenStack service;
Enter invalid characters as part of the Stack name 
[Expected result]
Error message in plain English
[Observed result] 
The technical service for the subscription OPENSTACK-K5(2) returned: The stack name 'Test-with-ü' is invalid. Valid characters are: alphanumerics, underscore, minus, point.
[Details]
See above; 
There is no 
minus: needs to dash
point: needs to be period 
alphanumericas is a slang: should be Alphanumeric Characters

Example:
The technical service for the subscription OPENSTACK-K5(2) returned: The stack name 'Test-with-ü' is invalid. Valid characters are: Alphanumeric Characters, underscore, dash, period.

[Messages and logs]

**Comment by MiethanerR** [comment]
corrected the translation in the message properties
